analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ProgramStateTest.cs

```diff
@@ -141,6 +142,25 @@ public void SetOperationValue_Overrides()
             sut[operation].Should().Be(value2);
         }
 
+        [TestMethod]
+        public void SetOperationValue_Null_ReturnsNull()
+        {
+            var operation = TestHelper.CompileCfgBodyCS("var x = 42;").Blocks[1].Operations[0];
+            var sut = ProgramState.Empty;
+
+            sut[operation].Should().BeNull();
+            sut = sut.SetOperationValue(operation, null);
+            sut[operation].Should().Be(null);
```

Comment:
would be nice to be consistent:
```suggestion
            sut[operation].Should().BeNull();
```